### DISCUSSION TITLE
python language not working

### DISCUSSION THREAD

**Discussion by iamheavymetalx7** [cpeditor/cpeditor#1159] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

it is not running,showing compile first but no option to compile the python file

### Expected Behavior

it should run the code

### Steps To Reproduce

1. open a file
2. use python as default language
3. write some code
4. press run

### CP Editor Version

6.10.3

### OS

macOS ventura

### Are you willing to contribute to this?

- [ ] I'm ready to fix this (and will open a PR soon)
- [X] I'd like to have a try (with the help of the maintainers)
- [ ] No, thanks

**Comment by ouuan** [comment]
Please check whether Python is properly installed and the environment variable for Python is set.

**Reply by iamheavymetalx7** [reply]
@ouuan Yes, it works perfectly in my vs code. I usually use python in VS code, do we have to set up everything in cp editor again?
